gradio/components.py

```diff
@@ -3993,9 +3990,24 @@ def update(
         }
         return updated_config
 
+    def _process_chat_messages(self, chat_message: str | Dict | None):
+        if isinstance(chat_message, str) and os.path.exists(chat_message):
```

Comment:
I'm confused how `chat_message` could ever be a `Dict`:
```suggestion
    def _process_chat_messages(self, chat_message: str | None):
        if chat_message and os.path.exists(chat_message):
```